.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Docker
```

Comment:
I'd suggest to put the release-related `build-xxx` files into a separate `release` subfolder.

Reply:
like the idea but got this error from gh:
```
invalid value workflow reference: workflows must be defined at the top level of the .github/workflows/ directory
```

Reply:
> like the idea but got this error from gh

Then I'd suggest to rename the files at least, giving them the "release-" prefix, so that it's clear what those jobs are for.